Python/symtable.c

```diff
@@ -2440,18 +2486,52 @@ symtable_visit_params(struct symtable *st, asdl_arg_seq *args)
 }
 
 static int
-symtable_visit_annotation(struct symtable *st, expr_ty annotation)
+symtable_visit_annotation(struct symtable *st, expr_ty annotation,
+                          struct _symtable_entry *parent_ste, void *key)
```

Comment:
Why do we need the `parent_ste` argument when the only caller always passes `st->st_cur`? I realize we need to preserve a pointer to `parent_ste` because we are going to push a block, but that can be done internal to the function.